src/docfx/build/page/BuildPage.cs

```diff
@@ -178,14 +178,25 @@ private static (List<Error> errors, Schema schema, OutputModel model)
             LoadSchemaDocument(Context context, List<Error> errors, JToken token, Document file, Action<Document> buildChild)
         {
             var obj = token as JObject;
-            if (file.Schema is null)
+            if (file.Schema is null || !File.Exists($"data/{file.Schema.Type.Name}.json"))
```

Comment:
This looks like a hack that relying on that you have not removed strong model schemas. Will your TODO "remove strong model schemas" include refining this?

Reply:
Currently validation and transform are not relying on strong model.  The only reason to keep strong model is the Razor part.